example_test.go

```diff
@@ -913,12 +917,51 @@ func ExampleFuture_GetIterator() {
 }
 
 func ExampleConnect() {
-	conn, err := tarantool.Connect("127.0.0.1:3013", tarantool.Opts{
-		Timeout:     5 * time.Second,
-		User:        "test",
-		Pass:        "test",
-		Concurrency: 32,
-	})
+	ctx, cancel := context.WithTimeout(context.Background(), 500*time.Millisecond)
+	defer cancel()
+
+	conn, err := tarantool.Connect(ctx, "127.0.0.1:3013",
+		tarantool.Opts{
+			Timeout:     5 * time.Second,
+			User:        "test",
+			Pass:        "test",
+			Concurrency: 32,
+		})
+	if err != nil {
+		fmt.Println("No connection available")
+		return
+	}
+	defer conn.Close()
+	if conn != nil {
+		fmt.Println("Connection is ready")
+	}
+	// Output:
+	// Connection is ready
+}
+
+func ExampleConnect_reconnects() {
+	ctx, cancel := context.WithTimeout(context.Background(), 500*time.Millisecond)
+	defer cancel()
```

Comment:
The context will expire after 500ms, but a Reconnect timeout is 5 seconds.

So there is wiil be only one try in fact.

Reply:
Updated.